src/components/ChattyLLM/InputArea.vue

```diff
@@ -27,18 +28,28 @@
 					<SendIcon :size="20" />
 				</template>
 			</NcButton>
+			<AudioRecorderWrapper v-else
+				v-model:is-recording="isRecording"
+				:compact="true"
```

Comment:
wdyt of making the button large so it is more visible?
The switch from audio to text when someone types would be more drastic but can be solved with a css transition effect.